docs/source/tooling/remix.rst

```diff
@@ -0,0 +1,64 @@
+.. _tools.remix:
+
+Remix
+=====
+
+MythX is available as a plugin for `Remix <https://remix.ethereum.org>`_, a popular web-based IDE for smart contract development and deployment, created and hosted by the Ethereum Foundation.
+
+.. image:: img/remix.jpg
+
+Setup
+-----
+
+.. note:: These instructions will show the Remix interface that is current as of mid-2019. We recommend everyone use this interface.
+
+Because Remix is a web-based interface, no local installation in required. However, MythX will need to be activated specifically before use.
```

Comment:
"However, MythX needs to be activated from Remix plugins ecosystem list specifically before use" or something along those lines would read better I think.

Reply:
Thinking: _However, MythX will need to be specifically activated from within the Remix Plugin Manager before use._

Reply:
sounds good